Source/Scene/Composite3DTileContent.js

```diff
@@ -0,0 +1,305 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Core/getMagic',
+        '../ThirdParty/when'
+    ], function(
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        DeveloperError,
+        getMagic,
+        when) {
+    'use strict';
+
+    /**
+     * Represents the contents of a
+     * {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/TileFormats/Composite/README.md|Composite}
+     * tile in a {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/README.md|3D Tiles} tileset.
+     *
```

Comment:
Link back to the base class,  and use inheritdoc below (same comments as Batched3DModel3DTileContent